src/test/java/com/slisenko/education/xml/XMLParserTest.java

```diff
@@ -32,10 +32,10 @@ public void parseStr() {
 
         for (int i = 0; i < 3; i++) {
             student = new Student();
-            student.setName("Имя " + (3 - i));
-            student.setSurname("Фамилия " + (3 - i));
+            student.setName("Name" + (3 - i));
+            student.setSurname("Surname" + (3 - i));
             student.setYear(1990  + (3 - i));
-            student.setFaculty("Факультет " + (3 - i));
+            student.setFaculty("Faculty" + (3 - i));
             student.setCourse(3 - i);
             expected.add(student);
         }
```

Comment:
Rewrite test on data sorting to sort using different input cases

Reply:
I don't understand a little. No sorting is checked in the class XMLParserTest.